docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -93,6 +93,22 @@ Using a filter you retain all information::
 General aggregate functions
 ---------------------------
 
+.. function:: any_value(x) -> [same as input]
+
+    Returns an arbitrary non-null value from all the rows of column ``x``. This
+    allows you to return values for columns that are not directly part of the
+    aggregation in a query. For example, the following query returns the
+    customer name, while the aggregation is using the unique customer
```

Comment:
What does it mean that “the aggregation is using the unique customer identifier”. It’s not very clear.